ani-cli

```diff
@@ -513,7 +517,10 @@ trackma_title="$(printf '%s' "$anime_id" | tr '-' ' ' | awk '{for(i=1;i<=NF;i++)
 trap 'printf "\033[0m";[ -f "$logfile".new ] && rm "$logfile".new;exit 1' INT HUP
 
 # default options
-player_fn="mpv" #video player needs to be able to play urls
+case "$OSTYPE" in
+    darwin*) player_fn="iina";; #MacOS specific
```

Comment:
imo it would be preferable to check for the existence of iina in /Applications/ as well, since some people might already have iina installed, but not through homebrew; you can check how i used an or statement for syncplay to achieve that